user_data/strategies/Strategy002.py

```diff
@@ -16,7 +16,7 @@
 
 
 class Strategy002(IStrategy):
-    controller = StrategyController('Freq_Strategy002')
+    controller = StrategyController('Freq-Strategy002')
```

Comment:
revert, this needs to match the brain name constructed

Reply:
for some reason this is creating a 400 response from telegram bot

Reply:
there was a code in the notifier of execution replacing _ with space, that's the fix